packages/utils/npm-packages-exists.test.js

```diff
@@ -15,15 +14,4 @@ describe("npmPackagesExists", () => {
 	test("throws a TypeError when an npm package name doesn't include the prefix", () => {
 		expect(() => npmPackagesExists(["my-webpack-scaffold"])).toThrowError(TypeError);
 	});
-
-	test("resolves packages when they are available on npm", done => {
```

Comment:
I've removed this, probably causes a delay and hangs the CI. Useless as the file is fragile when cross-compiling ts/js